receiver/zipkin/proto_parse_test.go

```diff
@@ -102,9 +102,10 @@ func TestConvertSpansToTraceSpans_protobuf(t *testing.T) {
 	zi := new(ZipkinReceiver)
 	hdr := make(http.Header)
 	hdr.Set("Content-Type", "application/x-protobuf")
+	r := &http.Request{Header: hdr}
```

Comment:
If we keep the multiplexing in ServeHTTP while cleanly separating the methods for v1 and v2, then we don't have to manually create this Request or make this line change and make a specific unit test for v1.

If we want to ensure that they properly multiplex, we can just use ServeHTTP in tests like this
```go
req := http.NewRequest("POST", "http://opencensus.io/", blob)
req.Header.Set("Content-Type", CONTENT_TYPE)
rec := httptest.Recorder()
zi.ServeHTTP(rec, req)
res := rec.Result()
zBlob, _ := ioutil.ReadAll(res.Body)
zreqs, err := json.Unmarshal(zBlob, &recv)
```